zscaler/README.md

```diff
@@ -131,7 +131,7 @@ Need help? Contact [Datadog support][7].
 
 [1]: https://www.zscaler.com/products/zscaler-internet-access
 [2]: https://docs.datadoghq.com/agent/kubernetes/integrations/
-[3]: https://github.com/DataDog/integrations-core/blob/master/zscaler/datadog_checks/zscaler/data/conf.yaml.example
+[3]: https://github.com/DataDog/integrations-extras/blob/master/zscaler/datadog_checks/zscaler/data/conf.yaml.example
```

Comment:
Same as previous: This link 404's. I don't see any file like this in the zscaler directory.

Reply:
same, fixed! Thanks